app/javascript/gobierto_dashboards/webapp/containers/contracts/Summary.vue

```diff
@@ -169,13 +188,26 @@ export default {
   created() {
     this.items = this.buildItems();
     this.columns = assigneesColumns;
+
+    EventBus.$on('filtered-items-grouped', (data, value) => this.updateItems(data, value))
```

Comment:
I saw before sometimes this event could be sent with no `value`. You should provide some defaults when that happens